src/txmempool.cpp

```diff
@@ -247,6 +247,24 @@ void CTxMemPool::removeConflicts(const CTransaction &tx, std::list<CTransaction>
     }
 }
 
+void CTxMemPool::removeExpired(unsigned int nBlockHeight)
```

Comment:
Should this be int?  At the call site, pindexnew->nHeight is an int.

Reply:
Hmm... everywhere else in `CTxMemPool`, heights are unsigned. I would say stick with this for now, and we can revisit later.